.github/workflows/macos.yaml

```diff
@@ -168,6 +173,14 @@ jobs:
         # Note: we silence the output of this command because it contains metadata about the certificates.
         security set-key-partition-list -S apple-tool:,apple: -s -k "$KEYCHAIN_PWD" "build.keychain" &>/dev/null
 
+    - name: Cache Ruby dependencies required by Omnibus
+      uses: actions/cache@v4
+      with:
+        path: ~/go/src/github.com/DataDog/datadog-agent/omnibus/vendor/bundle
+        key: omnibus-deps-${{ hashFiles('_omnibus_cache_key_files/*') }}
+        restore-keys: |
+          omnibus-deps-
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What's the purpose of `|` ?

Reply:
I was mostly just following [the examples in the docs](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#input-parameters-for-the-cache-action) here. `|` is the block indicator for YAML for a multiline string where newlines are kept; in this case with a single element it probably doesn't matter but it doesn't hurt either.